docs/premium-features/break-the-monolith.md

```diff
@@ -6,61 +6,64 @@ slug: /break-the-monolith
 
 # Break The Monolith
 
-Amplication's new "Break The Monolith" feature helps you modernize monolithic architectures and transform them into microservices. Using AI, Amplication simplifies the transition of your entities to different services, streamlining your shift to a microservices architecture.
+Amplication's "Break The Monolith" feature helps you modernize monolithic architectures and transform them into microservices. It introduces a new drag-and-drop interface to simplify the process of moving entities across different services.
```

Comment:
Since this is the intro of the feature- I wouldn't focus here about the AI or the drag-and-drop interface. But mainly talk about the interface/way to simplify the process of breaking a monolith to create a modern architecture which is easier to maintain, scale etc. Then- we can discuss the ways we supply in order to achieve it